src/server/graphql/models/Organization/addBilling/addBilling.js

```diff
@@ -45,6 +51,7 @@ export default {
     const {creditCard, stripeId, stripeSubscriptionId, periodEnd, periodStart, orgUsers} = await r.table('Organization')
       .get(orgId)
       .pluck('creditCard', 'orgUsers', 'periodEnd', 'periodStart', 'stripeId', 'stripeSubscriptionId');
+    const segmentTraits = getUserSegmentTraits(userId);
```

Comment:
looks like you're saying segmentTraits is a promise, not the result of a promise